dist/parser/csv.js

```diff
@@ -102,7 +102,7 @@ var Parser = function () {
         var rows = [];
         var fillAndPush = function fillAndPush(row) {
           return rows.push(row.map(function (col) {
-            return col || '';
+            return col != null ? col : '';
```

Comment:
Why do you commit the `dist`?

Reply:
Not sure as well, @AckerApple did this way. 
But there is no need to modify the dist. we always build before publishing to npm

Reply:
I used `npm run build:dist` to regenerate the dist. Afterwards the only difference was mine so I assumed the expectation was to commit it.

Does this need to be removed or is it ok?

Reply:
Its ok :)